lib/node/valueExpression.ts

```diff
@@ -71,6 +82,30 @@ const ternaryMethod = (operator: string, separator: string) => {
     };
 };
 
+const prefixUnaryMethod = (operator: string) => {
+    return function(this: INodeable) {
+        return prefixUnaryOperator(operator)(this)
```

Comment:
This creates a new function every time a prefix unary operator is called which would impact performance if the operator is called in a loop